src/fhir/PlanDefinition_cove.json

```diff
@@ -0,0 +1,438 @@
+      "id": "Day317Msg0",
+      "name": "Day 317 Message 0",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 317,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "14:37:59"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Thinking of you, {name} 🙂 I'm not always available but if you need to talk to someone, the National Hotline number is 988 & press 1 and the Veterans Crisis Line number is 838-255. -Riley"
+          }
+        }
+      ]
+    },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "Day371Msg0",
+      "name": "Day 371 Message 0",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 371,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "15:33:48"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Can you believe a year has gone by and I'm sending my last message from this program? I have truly enjoyed being in touch over the past year, {name} and remember that resources are always out there at wefacethefight.org. Know that I’m rooting for you! 💪"
+          }
+        }
+      ]
+    },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "Day371Msg1",
+      "name": "Day 371 Message 1",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 371,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "15:34:20"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Your final research survey should be headed your way in the next few days - we would appreciate your candid feedback about this program on the survey. -Riley"
+          }
+        }
+      ]
+    },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "NewYearsMsg",
+      "name": "New Years Message",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "event": [
+          "2025-01-01T17:21:16.000Z"
+        ]
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Sending you warm wishes for a bright year ahead! 🌟❄️🌟❄️ I am grateful to be in touch with you and hope this new year will be a good one, even as the seasons change -Riley"
```

Comment:
another `Riley` that is hard-coded perhaps? Should it be `{userName}`?

Reply:
per above